packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -59,23 +60,34 @@ export function getProxyForField(field: FlexTreeField): TreeNode | TreeValue | u
 		case FieldKinds.required: {
 			const asValue = field as FlexTreeTypedField<FlexFieldSchema<typeof FieldKinds.required>>;
 
-			// TODO: Ideally, we would return leaves without first boxing them.  However, this is not
-			//       as simple as calling '.content' since this skips the node and returns the FieldNode's
-			//       inner field.
-			return getOrCreateNodeProxy(asValue.boxedContent);
+			const maybeUnboxedContent = asValue.content;
+			if (maybeUnboxedContent === undefined) {
```

Comment:
If we are going to have separate code paths for value and required fields, then the required one should skip the undefined case.

Reply:
I don't see any change accounting for this comment, so I'm unresolving it.

Reply:
Woops, indeed forgot to remove it. Pushing without it now.